schema.sql

```diff
@@ -68,4 +68,54 @@ ALTER TABLE animals ADD COLUMN owner_id INT;
 ALTER TABLE animals ADD FOREIGN KEY(owner_id) REFERENCES owners(id);
 
 -- Update commited changes after making sure everything is fine
+COMMIT;
+
+-- DAY 4 Week 1 challenges
+-- Update create table called vets
+-- Start the trsnsaction
+BEGIN;
```

Comment:
- [ ] [OPTIONAL] Kindly remove the `BEGIN` and `COMMIT` keywords as there is no need for them because creating a table is not a transaction